examples/port-2-alibaba2-connection/outputs.tf

```diff
@@ -0,0 +1,26 @@
+output "alibaba_connection" {
+  value = module.create_port_2_alibaba_connection.primary_connection
+  sensitive = true
+}
+
+output "alibaba_connection_id" {
+  value = module.create_port_2_alibaba_connection.primary_connection_id
+}
+
+output "connection_name" {
```

Comment:
Redundant output. Not needed.

Reply:
Addressed with commit: [e2c6a1aff16a60d8cd9bdf5a6c1d4a2e5eb755cb](https://github.com/equinix/terraform-equinix-fabric/pull/155/commits/e2c6a1aff16a60d8cd9bdf5a6c1d4a2e5eb755cb)